src/azure-cli/HISTORY.rst

```diff
@@ -51,6 +51,10 @@ Release History
 * `az storage remove`: Change `--inlcude` and `--exclude` parameters to `--include-path`, `--include-pattern`, `--exclude-path` and`--exclude-pattern` parameters
 * `az storage sync`: Add `--include-pattern`, `--exclude-path` and`--exclude-pattern` parameters
 
+**SQL**
+
+* Update az sql mi update with two new properties tier and family.
```

Comment:
nit: please format this string little better. It sound weird: "Update az sql mi update". Also "properties: tier and family"

Reply:
Done. Thanks